examples/basic_mysql_connection/index.js

```diff
@@ -4,17 +4,28 @@ var mysql = require('mysql');
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
 
 var connection = mysql.createConnection({
-	host: config.host,
+	 host: config.host,
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
+					'SELECT * FROM todos WHERE Done=1',
```

Comment:
This works if you assume that `Done` can only be zero or one. What's the type of `Done` as shown in `DESCRIBE todos`? Could it be anything else?